src/assets/scripts/aircraft/AircraftStripView.js

```diff
@@ -277,15 +277,15 @@ export default class AircraftStripView {
     updateViewForApron(destinationText, hasAltitude, isFollowingSID) {
         this.$speed.addClass(SELECTORS.CLASSNAMES.RUNWAY);
         this.$heading.addClass(SELECTORS.CLASSNAMES.RUNWAY);
-        this.$heading.text(FLIGHT_MODES.APRON);
+        this.$heading.text(FLIGHT_MODES.APRON.toUpperCase() );
```

Comment:
I think this would be better handled by updating the constant value to be uppercase (which, in enums, the value _is_ usually uppercase).  Verify usages and make sure we aren't doing any direction comparisons with the enum values.  Basically, do a project search for each enum to verify usages to make sure you can make the value uppercase.